### PR TITLE
Move to using the 16.04 credentials docker container for testing.

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

**Comment by feanil** [comment]
@clintonb @rlucioni more movement to 16.04

File: .travis/run_tests.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -xe
+. /edx/app/credentials/venvs/credentials/bin/activate
+. /edx/app/credentials/nodeenvs/credentials/bin/activate
+
+apt update
+#apt install -y xvfb firefox gettext wget
```
Comment (rlucioni) [inline .travis/run_tests.sh]: @feanil can this be removed?

Comment (feanil) [inline .travis/run_tests.sh]: Yea, I'll remove this.

**Comment by feanil** [comment]
@rlucioni those were being changed when the translations tests run and got accidentally pulled in.  I've removed them.

**Pull request merged**